mne_bids/read.py

```diff
@@ -917,7 +919,11 @@ def get_head_mri_trans(bids_path, extra_params=None, t1_bids_path=None,
     # neuromag RAS coordinate system in order to compare the with MEG landmarks
     # see also: `mne_bids.write.write_anat`
     if fs_subject is None:
+        warn('Passing "fs_subject=None" has been deprecated and will raise '
```

Comment:
I see a typo above "We need to convert the to the" -> "We need to convert to the"

same with "compare the with MEG landmarks"

Reply:
Resolved